packages/primitives/src/Drawer.stories.tsx

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useEffect, useState } from "react";
+import { Portal } from "@ark-ui/react";
+import { Meta, StoryFn } from "@storybook/react";
+import { Cross } from "@ndla/icons/action";
+import { styled } from "@ndla/styled-system/jsx";
+import { Button, IconButton } from "./Button";
+import {
+  DialogBackdrop,
+  DialogBody,
+  DialogCloseTrigger,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogPositioner,
+  DialogRoot,
+  DialogRootProps,
+  DialogStandaloneContent,
+  DialogTitle,
+  DialogTrigger,
+  DrawerContent,
+  DrawerContentVariantProps,
+} from "./Dialog";
+import { Text } from "./Text";
+
+type StoryProps = DialogRootProps & DrawerContentVariantProps;
+
+/**
+ * En Drawer er en stylet versjon av `DialogContent`. På samme måte som `DialogContent` tilbys `DrawerContent` som en helper-komponent som kombinerer `DialogPositioner`, `DialogBackdrop` og `DialogContentStandalone`.
+ */
+const meta: Meta<StoryProps> = {
```

Comment:
Ser ikke ut som at autodocs satt veldig pris på at jeg satt sammen props fra to forskjellige komponenter. Kan eventuelt håndskrive typen til `position` så storyen funker ordentlig.

Reply:
Hadde jo vore fint å få valg på position også.